packages/itwinui-css/src/select/select.scss

```diff
@@ -153,18 +147,14 @@
   @include iui-select-tag-container;
 }
 
-.iui-select-tag {
+.iui-select-tag:is(.iui-tag) {
```

Comment:
It's mainly for increasing specificity, since `select.scss` is ordered before `tag.scss` and we want these styles to take preference.

And yes, we do want both classes present together. Think of `:is(.iui-tag)` as replacing `@include iui-tag`.